forms.rst

```diff
@@ -542,11 +542,11 @@ the guessed field.
 .. index::
    single: Forms; Field type guessing
 
-Field Type Options Guessing
+Field Attributes Guessing
```

Comment:
No I don't :) I was suggesting to add undocumented guessed options, not to remove these ;)